packages/client/package.json

```diff
@@ -37,7 +37,7 @@
     "@osdk/generator-converters": "workspace:~",
     "@osdk/internal.foundry.core": "workspace:~",
     "@osdk/internal.foundry.ontologiesv2": "workspace:~",
-    "@osdk/shared.client": "workspace:~",
+    "@osdk/shared.client": "workspace:^",
```

Comment:
All references for `@osdk/shared.client` should be `^` so that we have maximum change of cross SDK generation/platform sdks working together.

This is okay because we should basically not ever touch that package without extreme care.